### ISSUE TITLE
problem renaming items

### ISSUE THREAD

**Issue by dberardo-com** [lukasbach/react-complex-tree#8]
**Describe the bug**
Abort and submit rename operations seems to be broken

my situation:

- using controlled tree component
- implementing this functions:
```
 onStartRenamingItem={(item, name) =>
        console.log(`${item.data} being renamed to ${name}`)
      }
      onRenameItem={(item, name) => {
        console.log(`${item.data} renamed to ${name}`);
        handleRename(item, name);
      }}
      onAbortRenamingItem={(item, name) =>
        console.log(`${item.data} aborted to ${name}`)
      }
```

- output:

![image](https://user-images.githubusercontent.com/65530457/139216615-4650764b-f5f9-4902-9c05-706ece57a868.png)

two problems here:

1. clicking on the renaming-submit-button does not actually trigger the "onRenameItem" callback
2. pressing "escape" key while renaming item does not trigger onAbortRenamingItem
3.  clicking outside of the input text field while renaming item does not trigger onAbortRenamingItem

**Comment by lukasbach** [comment]
This is now fixed as of version v1.1.3 of this package. Please update your dependency to receive the fix.

If this issue or a similar problem persists, feel free to reopen this issue or create a new one. Thanks again for your issue report! :)

**Comment by lukasbach** [comment]
Working demo: https://rct.lukasbach.com/storybook/?path=/story/core-basic-examples--actions
